front/app/services/ideaCustomFieldsSchemas.ts

```diff
@@ -57,11 +57,26 @@ export interface IIdeaFormSchemas {
   };
 }
 
+const getInputFormsSchemaEndpoint = (
```

Comment:
I decided against using one function for this scenario and the scenario below because this code is old and it would be better not to link it to any new work directly